dev/com.ibm.ws.security.fat.common.backchannelLogout/fat/src/com/ibm/ws/security/backchannelLogout/fat/CommonTests/LogoutTokenValidationTests.java

```diff
@@ -599,6 +628,133 @@ public void LogoutTokenValidationTests_invalid_iat_1971() throws Exception {
+     */
+    @Test
+    public void LogoutTokenValidationTests_invalid_exp_1971() throws Exception {
+
+        long exp = 36010452; // 02/21/1971
+        long iat = exp - minutesToSeconds(2);
+
+        genericInvalidIatOrExpClaimTest(iat, exp);
+
+    }
+
+    /**
+     * Test that the backchannelLogout fails when we specify an exp that is from yesterday
+     *
+     * @throws Exception
+     */
+    @Test
+    public void LogoutTokenValidationTests_invalid_exp_yesterday() throws Exception {
+
+        long exp = System.currentTimeMillis() / 1000 - hoursToSeconds(24);
+        long iat = exp - minutesToSeconds(2);
+
+        genericInvalidIatOrExpClaimTest(iat, exp);
+
+    }
+
+    /**
+     * Test that the backchannelLogout fails when we specify an exp that is just in the past (just beyond the clockskew)
+     *
+     * @throws Exception
+     */
+    @Test
+    public void LogoutTokenValidationTests_invalid_exp_past_beyond_clockSkew() throws Exception {
+
+        long clockSkew = 5;
+        long exp = System.currentTimeMillis() / 1000 - minutesToSeconds(clockSkew);
+        long iat = exp - minutesToSeconds(2);
+
+        genericInvalidIatOrExpClaimTest(iat, exp);
+
+    }
+
+    /**
+     * Test that the backchannelLogout passes when we specify an exp that is the current time (due to the clowSkew)
+     *
+     * @throws Exception
+     */
+    @Test
+    public void LogoutTokenValidationTests_invalid_exp_now() throws Exception {
+
+        long exp = System.currentTimeMillis() / 1000;
+        long iat = exp - minutesToSeconds(2);
+
+        JWTTokenBuilder builder = loginAndReturnIdTokenData(defaultClient);
+        builder.setClaim(Constants.PAYLOAD_ISSUED_AT_TIME_IN_SECS, iat);
+        builder.setClaim(Constants.PAYLOAD_EXPIRATION_TIME_IN_SECS, exp);
+
+        String logutOutEndpoint = buildBackchannelLogoutUri(defaultClient);
+
+        List<endpointSettings> parms = createParmFromBuilder(builder);
+
+        List<validationData> expectations = vData.addResponseStatusExpectation(null, Constants.INVOKE_BACK_CHANNEL_LOGOUT_ENDPOINT, Constants.OK_STATUS);
+
+        invokeBcl(logutOutEndpoint, parms, expectations);
+
+    }
+
+    /**
+     * Test that the backchannelLogout passes when we specify an exp that is tomorrow
+     *
+     * @throws Exception
+     */
+    @Test
+    public void LogoutTokenValidationTests_invalid_exp_tomorrow() throws Exception {
+
+        long tomorrow = System.currentTimeMillis() / 1000 + hoursToSeconds(24);
+        
+        genericAddedUpdatedClaimsTest(Constants.PAYLOAD_EXPIRATION_TIME_IN_SECS, tomorrow);
+
+    }
+
+    /**
+     * Test that the backchannelLogout passes when we specify an exp that is way in the future
+     *
+     * @throws Exception
+     */
+    @Test
+    public void LogoutTokenValidationTests_invalid_exp_2035() throws Exception {
+
+        long feb_21_2035 = 2055696852;
+
+        genericAddedUpdatedClaimsTest(Constants.PAYLOAD_EXPIRATION_TIME_IN_SECS, feb_21_2035);
```

Comment:
I know the other tests hard code the value, but it'd still be better to programmatically set this. It's possible these tests could still be running in 2035 and I'd like to avoid a seemingly random hard failure.
```suggestion
        long futureTime = System.currentTimeMillis() / 1000 + hoursToSeconds(24 * 365 * 10);

        genericAddedUpdatedClaimsTest(Constants.PAYLOAD_EXPIRATION_TIME_IN_SECS, futureTime);
```